packages/oui-form-actions/README.md

```diff
@@ -0,0 +1,48 @@
+# Form Actions
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-form-actions
+  on-submit="$ctrl.submit()"
+  on-cancel="$ctrl.cancel()"
+></oui-form-actions>
+```
+
+## API
+
+| Attribute     | Type     | Binding | One-time Binding | Values                 | Default   | Description                      |
+| ----          | ----     | ----    | ----             | ----                   | ----      | ----                             |
+| on-submit     | function | &       |                  |                        |           | submit form action               |
+| on-cancel     | function | &       |                  |                        |           | cancel form action               |
+| submit-text   | string   | @?      | true             |                        | "Submit"  | submit button text               |
+| cancel-text   | string   | @?      | true             |                        | "Cancel"  | cancel button text               |
+| disabled      | boolean  | <?      |                  |                        | false     | disabled flag                    |
+
+## Examples
+
+### Default
+
+```html:preview
+<oui-form-actions
+  on-submit="$ctrl.submit()"
+  on-cancel="$ctrl.cancel()"
+></oui-form-actions>
+```
+
+### Custom naming
+
+```html:preview
+<div ng-init="$ctrl.disabled = false">
```

Comment:
(optional) The class `oui-doc-preview-only-keep-children` can be used to prevent the tag to be displayed in the doc...
But maybe we want to keep this to clarify what is done.

Reply:
Nice too. Gonna apply it.